tests/test_cli.py

```diff
@@ -60,57 +60,6 @@ def test_run_command_works_with_data_dir(self):
 
         assert CUSTOM_APP_RUN_MESSAGE in result.output
 
-    def test_package_logs_are_streamed_if_asked_for(self):
-        """ Test that logs from the main part of the package are sent to stderr if the CLI option is enabled. """
-        with tempfile.TemporaryDirectory() as temporary_directory:
-            with mock.patch("logging.StreamHandler.emit") as mock_log_handler_emit:
-                CliRunner().invoke(
-                    octue_cli,
-                    [
-                        "--show-twined-logs",
-                        "run",
-                        f"--app-dir={TESTS_DIR}",
-                        f"--twine={self.TWINE_FILE_PATH}",
-                        f'--data-dir={os.path.join(TESTS_DIR, "data", "data_dir_with_no_manifests")}',
-                        f"--output-dir={temporary_directory}",
-                    ],
-                )
-
-            mock_log_handler_emit.assert_called()
-
-            self.assertTrue(
-                any(
-                    "Showing package logs as well as analysis logs (the package logs are recommended for software "
```

Comment:
So this is what I meant about some logs going to the cloud, some logs from the actual package going back to the asker. Will this capability be brought back in?

Reply:
I think the capability as it was there didn't really work as intended in the end and might have been misguided by me as a way to get around the bugs resolved in this PR that I didn't know existed at the time. I was planning to start from scratch on streaming some logs from the cloud etc. but in a different PR. I can do it in this one if you prefer?

Reply:
Happy for it to be separate